README.md

```diff
@@ -244,6 +245,7 @@ for detailed explanation and flags of each command see help
 
 [Configuration]: https://github.com/aurora-is-near/relayer2-public#how-to-configure
 [NEAR CLI]: https://docs.near.org/docs/tools/near-cli
+[Standalone Aurora Relayer and Refiner]: https://github.com/aurora-is-near/standalone-rpc
```

Comment:
JFI, there is another placeholder already defined for this purpose, please see `[Aurora Relayer v2 Dist]: TODO` and where it is used. 
if you prefer we can replace TODO with standalone-rpc link, or remove that part and use the wording in this PR